hardware/Layout-Desing-Guide.md

```diff
@@ -0,0 +1,486 @@
+
+<p align="center">
+  <img src="files/Layout-Design-Guide/crystal.png">
+</p>
+
+<div align="center">
+
+<b>Figure 18.</b> 
+
+</div>
+
+2. Connect the crystal case to the ground using many vias to avoid radiation of the ungrounded parts. Do not leave any metal unconnected and float that may be an unwanted radiator. 
+
+<p align="center">
+  <img src="files/Layout-Design-Guide/crystal2.png">
+</p>
+
+<div align="center">
+
+<b>Figure 19.</b> 
+
+</div>
+
+3. Use an isolating ground metal between the crystal and VDD traces to avoid any detuning effects on the crystal caused by the nearby power supply, and to avoid the leakage of the crystal or clock signal and its harmonics to the supply lines.
+
+<p align="center">
+  <img src="files/Layout-Design-Guide/crystal3.png">
+</p>
+
+<div align="center">
+
+<b>Figure 20.</b> 
+
+</div>
+
+****
+## 3.4. Layout Design Practices - VDD filtering
+
+1. The lowest value capacitors have to be placed the closest to the VDD pins. besides that, they need good grounding (with many close vias) at the ground side. The largest value capacitor filters can be placed far away from the VDD pins and is not required in case of battery operation.
+
+
+
+<p align="center">
+  <img src="files/Layout-Design-Guide/VDD.png">
+</p>
+
+<div align="center">
+
+<b>Figure 21.</b> 
+
+</div>
+
+***
+
+## 3.5. Layout Design Practices - General
+
+1. Use 50 Ohm grounded coplanar transmission lines to connect distant RF components. Online impedance calculators can be used to get the necessary trace width and gap tp ground copper,  as figure below shows the typical values for 50 Ohm grounded coplanar lines. 
+
+<p align="center">
+  <img src="files/Layout-Design-Guide/coplanar2.png">
+</p>
+
+<div align="center">
+
+<b>Figure 22.</b> 
+
+</div>
+
+<p align="center">
+  <img src="files/Layout-Design-Guide/coplanar.png">
+</p>
+
+<div align="center">
+
+<b>Figure 23.</b> 
+
+</div>
+
+2. The grounding effect in the thermal straps used with capacitors should be improved. In addition, the trace near the GND pin of these capacitors should be thickened, this will minimize series parasitic inductance between the ground pour and the GND pins. Additional vias placed close to the GND pin of capacitors (thus connecting it to the bottom or inner layer GND plane) will further help reduce these effects.
+
+<p align="center">
+  <img src="files/Layout-Design-Guide/vias.png">
+</p>
+
+<div align="center">
+
+<b>Figure 24.</b> 
+
+</div>
+
+3. The exposed pad footprint for the paddle of the EFR32 IC should use as many vias as possible to ensure good grounding and heat sink capability. In the reference designs there are 25 vias for the 7x7 mm sized package ICs, each with 10 mil diameter. The paddle ground should also be connected to the top layer GND metal, if possible, to further improve RF grounding; this may be accomplished with diagonal trace connections through the corners of the EFR32 IC footprint.
```

Comment:
"this may be accomplished with diagonal trace connections through the corners of the EFR32 IC footprint."  I'd be concerned about trace-to-trace design rule violations in routing like this.  Are we sure this is okay with our QFN package footprints?

Reply:
You are right, it's not true in case of QFN package. I've removed this sentence.